internal/cmdrun/va/list/list.go

```diff
@@ -0,0 +1,99 @@
+package list
+
+import (
+	"context"
+	"strconv"
+	"time"
+	"vf-admin/internal/api"
+	"vf-admin/internal/utils"
+
+	openapi_types "github.com/deepmap/oapi-codegen/pkg/types"
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+// CmdRunE is what's executed when running `vf-admin va list`
+func CmdRunE(cmd *cobra.Command, args []string) error {
+	flags := cmd.Flags()
+	postcode, _ := flags.GetString("postcode")
+
+	var minDate *openapi_types.Date
+
+	if flags.Changed("mindate") {
+		t, _ := flags.GetString("mindate")
+		t2, tErr := time.Parse("2006-01-02", t)
+		if tErr != nil {
+			color.Red(tErr.Error())
+			return nil
+		}
+		minDate = &openapi_types.Date{Time: t2}
+	}
+
+	params := api.ListVaccineAvailabilityApiV1VaccineAvailabilityGetParams{PostalCode: postcode, MinDate: minDate}
+
+	// Create the API client
+	client, cErr := utils.GetAPIClient()
+	if cErr != nil {
+		color.Red(cErr.Error())
+		return nil
+	}
+
+	// Create spinner
+	spinner, sErr := utils.GetDefaultSpinnerForHTTPOp("list", "got", "va")
+	if sErr != nil {
+		color.Red(sErr.Error())
+		return nil
+	}
+	_ = spinner.Start()
+
+	res, rErr := client.ListVaccineAvailabilityApiV1VaccineAvailabilityGetWithResponse(context.Background(), &params)
+
+	if rErr != nil {
+		spinner.StopFailMessage(rErr.Error())
+		_ = spinner.StopFail()
+		return nil
+	}
+
+	if res.StatusCode() != 200 {
+		spinner.StopFailMessage(res.Status() + ": " + string(res.Body))
+		_ = spinner.StopFail()
+		return nil
+	}
+
+	_ = spinner.Stop()
+
+	colNames := []string{"id", "date", "number available", "number total", "vaccine", "input type", "tags", "location id", "location name", "location line 1", "location line 2", "location province", "organization", "created at"}
```

Comment:
It would be nice to put everything about `location` in one column because the table gets stretched too far out if the screen isn't wide enough